src/lib/LED/devRGB.cpp

```diff
@@ -18,9 +33,18 @@ void WS281Binit()
 
 void WS281BsetLED(uint32_t color)
 {
-    strip.ClearTo(RgbColor(HtmlColor(color)));
+    strip.SetPixelColor(STATUS_LED_NUMBER, RgbColor(HtmlColor(color)));
     strip.Show();
 }
+
+void setStripColour(RgbColor color)
```

Comment:
This should follow the naming similar to others like `WS281BsetStrip`.
Argument is also `RgbColor` and might be better to use `uint32_t` and convert it internally. That would help to maintain LED controlling in the future.